### ISSUE TITLE
[Solved] ParseGeoPoint doesn't work

### ISSUE THREAD

**Issue by ScanEat** [phillwiggins/flutter_parse_sdk#43]
Hi,

I have an object (DietPlan) which have a ParseGeoPoint (with real GPS location) but when I retrieve my object, there is a problem with the GeoPoint, I can't have the informations.

My full console :

```
Payload: {"className":"Diet_Plans","objectId":"lxeHxqnp8A","createdAt":"2019-01-16T09:30:19.147Z","updatedAt":"2019-01-21T15:01:39.774Z","Fat":10,"Name":"iet",","location":"{'__type': 'Pointer', className: GeoPoint, objectId: null}"}
```

The issue :
```
"location":"{'__type': 'Pointer', className: GeoPoint, objectId: null}
```

Thanks in advance

**Comment by phillwiggins** [comment]
A fix for this has been added to release v1.0.10. This will be released in the next few days.

**Comment by phillwiggins** [comment]
Sorted. Thanks
